templates/gitlab.yml

```diff
@@ -74,4 +78,4 @@ functions:
       - github.yml
     secrets:
       - payload-secret
-      - gitlab-api-token
\ No newline at end of file
+      - gitlab-api-token
```

Comment:
This should not be present. It will cause GitHub to fail

Reply:
This is shown as change due to my editor removing the newline. Otherwise haven't added the field, currently we are using this secret `gitlab-api-token` in the official `templates/gitlab.yml`, does this breaks things currently ?

Reply:
My mistake, I think you're right.